spec/loadbalancer/README.md

```diff
@@ -1,4 +1,4 @@
-## EC Load Balancer Spec
+## EC Gateway Load Balancer Spec
 ### Design
```

Comment:
@ramaraosrikakulapu Please add **Feature**, and **Goal** section as below example- 

## Goal
E.g. Load balancing traffic for EC connectivity as service.

## Feature
Cloud platform-agnostic, etc.

Reply:
@ramaraosrikakulapu proposed ```Goal``` looks good. Still need content for the ```Feature``` . And please help add ```Runtime Requirement``` to the readme.